modules/sharedaddy/sharing-sources.php

```diff
@@ -865,9 +865,7 @@ public function display_footer() {
 
 class Share_Facebook extends Sharing_Source {
 	public $shortname = 'facebook';
-	public $genericon = '\f204';
-	private $share_type = 'default';
```

Comment:
Was this an accidental deletion when doing a merge?  Doublecheck the wpcom source and confirm.